README.md

```diff
@@ -15,12 +32,10 @@ Prerequisites
 You will need a Chromebook with developer mode enabled.  To do so, select your device on
 [the ChromiumOS Wiki](https://www.chromium.org/chromium-os/developer-information-for-chrome-os-devices) and follow the instructions listed there.
 
-Please be aware of the fact that developer mode is insecure if not properly configured. Setting a password as instructed in the VT-2 login screen is essential. It is also recommended to enable signed boot:
+Please be aware of the fact that developer mode is insecure if not properly configured. Setting a password as instructed in the VT-2 login screen is recommended. It is also recommended to enable signed boot:
 
-```bash
-sudo chromeos-setdevpasswd
-sudo crossystem dev_boot_signed_only=1
-```
+    sudo chromeos-setdevpasswd
+    sudo crossystem dev_boot_signed_only=1
```

Comment:
Better to not indent for easier copy/pasta.

Reply:
It shows the same thing as backticks (the extra space will not be shown):
```
This is using backticks
```
    This is using indent

Reply:
This is a code block though so it should have backticks.